docs/repo-docs/reference/configuration.mdx

```diff
@@ -117,6 +117,47 @@ To help with incremental migration or in situations where you can't use the `pac
   environment variable.
 </Callout>
 
+### `cacheDir`
+
+Default: `".turbo/cache"`
+
+Specify the filesystem cache directory.
+
+```jsonc title="./turbo.json"
+{
+  "cacheDir": ".turbo/cache"
+}
+```
+
+### `daemon`
+
+Default: `false`
+
+Turbo can run a background process to pre-calculate values used for determining work that needs to be done. This standalone process (daemon) is an optimization, and not required for proper functioning of turbo.
+
+```jsonc title="./turbo.json"
+{
+  "daemon": true
+}
+```
+
+### `envMode`
+
+Default: `"strict"`
+
+Turborepo's Environment Modes allow you to control which environment variables are available to a task at runtime:
```

Comment:
In general, we like to provide the link the first time we mention a term. The game I like to play is: Imagine your coworker just linked you this page and you're only glancingly aware that Turborepo is in your company's codebase. You're not terribly familiar with Turborepo but you're here to learn more about it.

Putting yourself in those shoes, you're likely to read this "Environment Modes" term and think "Uh, what's that?", and want to be linked to more information in that moment. You could, of course, keep reading and make it to the end of this section, but you'd probably appreciated the link sooner.

So, that said, I'd be lifting the link that's a few lines below to here, and removing the sentence on 153. Both a context and conciseness win!

Reply:
this was lifted verbatim from the [existing docs](https://github.com/vercel/turborepo/blob/219c1140d781a7bd5657e7d718e5e96247c8b00c/docs/repo-docs/crafting-your-repository/using-environment-variables.mdx?plain=1#L84-L87) - it seems strange, in this case, to link to other docs when the place the link goes says exactly the same thing that's here.  would you like to just link it anyway?

Reply:
Ah, heard! Ignore then. 😄